src/pages/index.js

```diff
@@ -93,22 +203,72 @@ const editProfilePopup = new PopupWithForm({
 });
 editProfilePopup.setEventListeners();
 
+profileEditBtn.addEventListener("click", () => {
+  const userData = userInfo.getUserInfo();
+  profileInputTitle.value = userData.title;
+  profileInputDescription.value = userData.description;
+  profileEditFormValidator.resetValidation();
+  editProfilePopup.open();
+});
+
+// AVATAR MODAL AND METHODS -----
+
+const handleAvatarFormSubmit = (data) => {
+  function makeRequest() {
+    return api.updateProfileAvatar(data.link).then(() => {
+      userInfo.changeAvatar(data.link);
+      editAvatarPopup.close();
```

Comment:
` editAvatarPopup.close();` is not needed if you use `handleSubmit`, because it contains the closing